### PR TITLE
Button guidelines order

### PR DESCRIPTION
**Pull request by kpoulincoveo**
Updated the guidelines related to the order of the buttons that appear in the header.

### Proposed Changes

After much investigation, we came to the decision that that the primary button for this group of button should remain on the left side of the group of buttons.
The principal reason for this change of guidelines is that the we wanted to make sure that the order of actions was consistent with their order importance and the logical reading order.
Since we have a couple of occurrences where the header contains additional secondary actions grouped under the […] (more/actionable items) button, having that button on the right felt wrong.

Read the detailed guidelines here for additional details: https://coveord.atlassian.net/wiki/spaces/UX/pages/3063545872/Header+button+ordering

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/docs/Button.md
Comment (toofff) [inline packages/website/docs/Button.md:1]:
```suggestion
# Best Practices
```

File: packages/website/docs/Button.md
Comment (toofff) [inline packages/website/docs/Button.md:26]: Is it possible to have several primary buttons in the same page?

**Review by gdostie — APPROVED** [review]

File: packages/website/docs/Button.md
Comment (jlevesquecoveo) [reply packages/website/docs/Button.md:26]:
```suggestion
In a [page header](https://plasma.coveo.com/layout/PageHeader), **the primary button should appear on the left** and the secondary button should be to the right of the primary button. The primary action usually relates to the creation of new elements, while secondary actions are usually used to trigger troubleshooting or management actions, such as activity review. If multiple secondary actions are required, group them using an [actionable item](https://plasma.coveo.com/form/ActionableItem) on the rightmost side of the header. 
```

**Review by jlevesquecoveo — APPROVED** [review]

**Comment by kpoulincoveo** [comment]
@gdostie ready to merge! 😄

**Comment by gdostie** [comment]
> @gdostie ready to merge! 😄

Let's wait for one more dev approval 🙌

**Comment by aiheon** [comment]
> > @gdostie ready to merge! 😄
> 
> Let's wait for one more dev approval 🙌

You called for me? 👀 :trollface:

**Review by aiheon — APPROVED** [review]

**Pull request merged**
